library/lib.sh

```diff
@@ -164,6 +164,14 @@ lsrGetCollectionPath() {
     fi
 }
 
+lsrRunCmdUntilSuccess() {
```

Comment:
There is a beakerlib `rlWaitForCmd` which will run command for specified number of times, or specified number of seconds https://github.com/beakerlib/beakerlib/blob/master/src/synchronisation.sh#L251 - can you use this instead?

Reply:
Good point, I applied this